### DISCUSSION TITLE
Multiple External Login Providers

### DISCUSSION THREAD

**Discussion by ajahangard** [oqtane/oqtane.framework#4844] [Q&A]
Is it possible to have multiple external login providers for a single Oqtane site? for example enabling Facebook, Google and Github as external login providers simultaneously?

**Comment by sbwalker** [comment]
No, it is not currently possible to have multiple external providers in a site

**Reply by ajahangard** [reply]
What is the recommended way to add this functionality?

**Reply by sbwalker** [reply]
@ajahangard it would require a fairly significant enhancement to the framework
